parent/pom.xml

```diff
@@ -119,7 +119,7 @@
         <cxf.geronimo.transaction.version>3.1.4</cxf.geronimo.transaction.version>
         <cxf.geronimo.openapi.version>1.0.12</cxf.geronimo.openapi.version>
         <cxf.glassfish.json.version>1.0.4</cxf.glassfish.json.version>
-        <cxf.guava.version>20.0</cxf.guava.version>
+        <cxf.guava.version>28.2-jre</cxf.guava.version>
```

Comment:
@davidkarlsen Unfortunately `guava` is problematic to update :( The OSGi bundles are failing to deploy with Guava > 20.0, we need to update the affected dependencies before that:
```
 [caused by: Unable to resolve org.apache.servicemix.bundles.opensaml/3.4.3.1: missing requirement [org.apache.servicemix.bundles.opensaml/3.4.3.1] osgi.wiring.package; filter:="(&(osgi.wiring.package=com.google.common.base)(version>=19.0.0)(!(version>=21.0.0)))"
```
:(

Reply:
@jbonofre - Could we have new opensaml bundles that allow a wider guava range?

Reply:
Also, note that for WSS4J we need to add the following to the feature: install -s mvn:com.google.guava/failureaccess/1.0.1

Reply:
:+1: thanks @coheigea

Reply:
Maybe this should be updated directly to 29.0?

Reply:
+1